From 9cb1667f9d3a9bcfc3b83466cd8d3b79f0554ff0 Mon Sep 17 00:00:00 2001
From: Azat Khuzhin <a3at.mail@gmail.com>
Date: Wed, 8 Jul 2015 01:37:09 +0300
Subject: [PATCH 1/3] fix typos in manpage

This patch is taken from
ftp://ftp.debian.org/debian/pool/main/b/blktool/blktool_4-7.debian.tar.xz

Upstream-Status: Inappropriate [upstream is dead]
Signed-off-by: Alexander Kanavin <alexander.kanavin@linux.intel.com>

---
 blktool.8 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/blktool.8 b/blktool.8
index a1f5c96..45b7724 100644
--- a/blktool.8
+++ b/blktool.8
@@ -191,7 +191,7 @@ Query or set device bus state (0 off, 1 on, 2 tristate)
 Query the detected (or overridden, via -t) device class.
 Typically this will result in 'ATA' or 'SCSI' for most devices.
 Detection is based on device major; thus your SATA device may appear as
-'SCSI'.
+\&'SCSI'.
 
 .TP
 .B cd-speed
@@ -237,7 +237,7 @@ Omitting the on/off argument will print the current state.
 
 .TP
 .B media
-Lock in (or unlock) a removeable device.
+Lock in (or unlock) a removable device.
 
 .TP
 .B multiple-count
-- 
2.1.4

